### DISCUSSION TITLE
Lost login function

### DISCUSSION THREAD

**Discussion by YegorFrolov** [requarks/wiki#6144] [Help / Questions]
02/08/2023 everything worked 02/09/2023 the possibility of login was gone

app.js?1675554523:486 TypeError: Cannot read properties of undefined (reading 'strategy')
    at a.filteredStrategies (login.js?1675554523:2:10961)
    at Wt (app.js?1675554523:486:11329)
    at hn.run (app.js?1675554523:486:27798)
    at mn (app.js?1675554523:486:25769)
    at Array.<anonymous> (app.js?1675554523:486:12371)
    at Zt (app.js?1675554523:486:11814)

![Снимок экрана 2023-02-09 в 12 20 41](https://user-images.githubusercontent.com/87973055/217770740-f2d9202b-6614-4960-ab2b-472099aaf8bc.png)

Please, tell what could be wrong and how to fix it

![Снимок экрана 2023-02-09 в 12 24 08](https://user-images.githubusercontent.com/87973055/217771298-3871a844-36ec-4c10-8278-cb3fd2b8da18.png)

**Comment by NGPixel** [comment]
Have you upgraded to another version between both dates?

**Comment by lzb-libra** [comment]
Hello, I have a rather tricky issue. Initially, after setting up the wiki, I used the administrator account to disable the login interface due to some reasons. However, due to a prolonged period without logging in, I am now only able to view content and cannot log in to edit my content. Is there any way I can display the login interface again?

**Reply by NGPixel** [reply]
Have you read the docs?

https://docs.requarks.io/auth#hide-local-provider

**Reply by lzb-libra** [reply]
Sorry, I haven't gone through the documentation specifically. I installed it based on the installation steps provided online.
